### PR TITLE
Added GPX exporter 

### PR DESCRIPTION
**Pull request by bonchan**
GPX files can be imported in google earth pro to show a 3d view of the flight to achieve something like [this](https://www.youtube.com/watch?v=4umx4QRSxHs). 

Instructions [here](https://www.youtube.com/watch?v=dhgQ8aPUq_U)

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: js/webworkers/gpx-export-worker.js
Comment (haslinghuis) [inline js/webworkers/gpx-export-worker.js:27]:
```suggestion
  for (const chunk of event.data.frames) {
    for (const frame of chunk) {
```

File: js/webworkers/gpx-export-worker.js
Comment (haslinghuis) [inline js/webworkers/gpx-export-worker.js:45]: Are you working on this otherwise leave a comment in source.

File: js/webworkers/gpx-export-worker.js
Comment (bonchan) [reply js/webworkers/gpx-export-worker.js]: done

File: js/main.js
Comment (haslinghuis) [inline js/main.js:921]: Missing semicolon ;

**Review by haslinghuis — DISMISSED** [review]

File: js/main.js
Comment (bonchan) [reply js/main.js]: done

**Review by haslinghuis — APPROVED** [review]

File: js/gpx-exporter.js
Comment (KarateBrot) [inline js/gpx-exporter.js:32]: Missing end of line (LF)

**Review by KarateBrot — CHANGES_REQUESTED** [review]

File: js/gpx-exporter.js
Comment (bonchan) [reply js/gpx-exporter.js]: done

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
@bonchan please rebase

**Review by McGiverGim — APPROVED** [review]

**Comment by haslinghuis** [comment]
Another reference for this awesome function: https://www.youtube.com/watch?v=uBXmUG6JZkc

**Comment by bonchan** [comment]
> @bonchan please rebase

done!

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
